tests/test_auth.py

```diff
@@ -0,0 +1,14 @@
+from unittest.mock import MagicMock, patch
+
+from src.auth import refresh_access_token
+
+
+class MockPost(MagicMock):
+    def json(self):
+        return {"access_token": "FAKE_TOKEN"}
+
+
+def test_refresh_access_token():
+    with patch("requests.post", MockPost):
```

Comment:
requests.post gets replaced by an instance of MockPost in the test
this allow the test to avoid making the http request and to have direct access to the json() method that returns a json object containing "access_token" as key